src/parallel/parallel.hpp

```diff
@@ -165,9 +169,12 @@ class ParallelRobinHoodHashTable {
     bool
     compare_and_set_key_val(size_t index, KeyValue prev_kv, KeyValue new_kv);
 
-    ParallelBucket
+    KeyValue
```

Comment:
Why do we do KeyValue rather than ParallelBucket?

Reply:
I want to ensure people understand this returns a copy, and don't try to call atomic operations on the return value directly (which wouldn't modify the original value).